site/docs/v1/tech/apis/_form-request-body.adoc

```diff
@@ -5,6 +5,13 @@
 [field]#form.data# [type]#[Object]# [optional]#Optional#::
 An object that can hold any information about the Form that should be persisted.
 
+[field]#form.type# [type]#[String]# [optional]#Optional# [default]#defaults to `registration`# [since]#Available since 1.20.0#::
+The type of registration associated with this form. The possible values are:
++
+ * `registration` which is used for self-service registration processes
+ * `adminRegistration` which is used for administrator registration processes, utilizes `registration.data.*` custom fields
+ * `adminUser` which is used for administrator registration process, utilizes `user.data.*` custom fields
```

Comment:
This form can be used to customize the add and edit User form in the FusionAuth UI.